### DISCUSSION TITLE
幫眼睛拍張照，就知道生什麼病──中國醫療新創顛覆看診

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#723] [General]
https://futurecity.cw.com.tw/article/85
新型整療

**Comment by Valentinaa0214** [comment]
這讓我想到之前很紅的Theranos血液檢測設備，操作方式是從指尖的幾滴血中搜集患者的大量數據來分析其病情。
想法雖然很好，但實際卻是完全無法實現，希望這種設備是真的能夠有實際且精確的效果，達到顛覆看診方式的目標。

**Comment by HsuChen88** [comment]
幫眼睛拍張照，就可以被AI給眼睛。
這個技術的突破等於是將所有眼睛相關的昂貴檢查都結合再一起了，省去時間、人力、金錢。
